### PR TITLE
Fix 'Shuffle All' for video playlist

### LINKED ISSUE #1712: "Shuffle All" and "Play From Here" not working
### Describe the bug

1. Create a playlist (I had to do this on my computer in the web interface, because I see no quick and easy way to turn an entire library into a playlist in the AndroidTV client, nor a way to play an entire library from the AndroidTV client).  I made a music video playlist of all my MV collection, only 261 items.
2. Load the playlist on a TV running the AndroidTV client
3. select "Shuffle All" option, and watch as the player proceeds to begin at the beginning of the playlist and play in order without any shuffling.
4. This is 3-Alternate, select any song in the list and choose "Play From Here" and watch as it proceeds to start again with the first song in the play list in the playlist's original order.

I expect the "Shuffle All" option to select a song at (semi-)random and continue playing all videos in the playlist as random.
I expect "Play From Here" to start playing the playlist in order from the song selected, but instead both options start at the beginning of the list in order.

### Logs

_No response_

### FFmpeg logs

_No response_

### Media info of the file

_No response_

### Application version

0.13.5

### Device information

Onn UHD Streaming Device (Onn 4K TV / askey sti6140d360)

### Android version

Android 10

### Jellyfin server version

10.7.7

### PR DESCRIPTION
**Pull request by kevgrig**
Fixes #1712

Signed-off-by: Kevin G <kevin@myplaceonline.com>

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/ItemListActivity.java [hunk 1]
```diff
@@ -499,7 +500,12 @@ private void play(List<BaseItemDto> items, int ndx, boolean shuffle) {
         PlaybackLauncher playbackLauncher = KoinJavaComponent.<PlaybackLauncher>get(PlaybackLauncher.class);
         if (playbackLauncher.interceptPlayRequest(this, items.size() > 0 ? items.get(0) : null)) return;
 
+        Timber.tag(ItemListActivity.class.getSimpleName()).d("play items: %d, shuffle: %b", items.size(), shuffle);
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/ItemListActivity.java:503]:
```suggestion
        Timber.d("play items: %d, shuffle: %b", items.size(), shuffle);
```

We never use the tag method

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/ItemListActivity.java (hunk 1 above)
Comment (kevgrig) [reply app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/ItemListActivity.java]: Updated and squashed

**Review by nielsvanvelzen — APPROVED** [review]

**Comment by kevgrig** [comment]
Thanks for approving! I don't have authority to merge - is that something you'll do or do I need to do some other steps for this?

**Comment by nielsvanvelzen** [comment]
I'll merge it when 0.14 is released. The master branch is currently frozen so we can merge critical fixes/translation updates and easily rebase the release branch on top of it.

**Pull request merged**